terraso_backend/apps/core/migrations/0024_landscape_profile_image.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.1 on 2022-11-07 15:24
```

Comment:
issue: this migration looks unrelated to area calculations and probably belongs in another branch.

Reply:
Yes, will figure out why these here and get rid of those commits.